.rspec

```diff
@@ -1,3 +1,4 @@
 --format documentation
 --color
 --require spec_helper
+--order random
```

Comment:
Just curious, why is it that we are setting this here? shouldn't it be an env variable?

Reply:
this is to make `rspec` run the tests in random order